tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Ordering Party Update .feature

```diff
@@ -76,30 +76,72 @@ Scenario: 2. Updating an ordering party with boundary values results in persiste
     And the order with orderId C000014-01 has LastUpdated time present and it is the current time
 
 @4616
-Scenario: 3. A non existent orderId returns not found
-    When the user makes a request to retrieve the ordering-party section with the ID INVALID
-    Then a response with status code 404 is returned
+Scenario: 3. Updating an ordering party, with a non existent model returns not found
+    When the user makes a request to update the order party with order ID C000014-01 with no model
+    Then a response with status code 400 is returned
 
 @4616
-Scenario: 4. If a user is not authorised then they cannot access the ordering-party
+Scenario: 4. If a user is not authorised then they cannot update the ordering-party

```

Comment:
Wondered whether it would be possible to simplify this test as the user is not logged in can we just send a simple request that doesn't contain all the information of an order record?

Reply:
👍